lib/fishjam_web/controllers/room_controller.ex

```diff
@@ -107,6 +107,10 @@ defmodule FishjamWeb.RoomController do
         room_id = Map.get(params, "roomId")
         {:error, :bad_request, "Cannot add room with id \"#{room_id}\" - room already exists"}
 
+      {:error, :room_doesnt_start} ->
+        room_id = Map.get(params, "roomId")
+        {:error, :bad_request, "Cannot add room with id \"#{room_id}\" - adding failed"}
```

Comment:
How about "unexpected error" instead? I feel like we're repeating ourselves with "adding failed"